core/src/wallet/cosmos/CosmosLikeAccount.cpp

```diff
@@ -357,14 +351,15 @@ namespace ledger {
                                                 if (operation.date <= upperDate) {
                                                         switch (operation.type) {
                                                                 case api::OperationType::RECEIVE: {
-                                                                        sum = sum + operation.amount;
+                                                                        sum = sum + operation.amount - operation.fees.getValueOr(BigInt::ZERO);
                                                                         break;
                                                                 }
                                                                 case api::OperationType::SEND: {
                                                                         sum = sum - (operation.amount + operation.fees.getValueOr(BigInt::ZERO));
                                                                         break;
                                                                 }
                                                                 default:
+                                                                        sum = sum - operation.fees.getValueOr(BigInt::ZERO);
                                                                         break;
```

Comment:
You can just do a `sum = sum - fees` outside from the switch

Reply:
Very true :D